src/regex_impl.hh

```diff
@@ -478,9 +478,10 @@ private:
         m_found_match = false;
         while (true) // Iterate on all codepoints and once at the end
         {
+            idle_func();
+
```

Comment:
Why did you change this ? I would expect that running a select syscall after every codepoint during regex matching is going to slow down things quite a bit. Doing it every 64k codepoints seemed regular enough for the cancellation use case, was that not the case ?

Reply:
yeah I misread "wrap" as "wrap around buffer"